baremetal/metal3data_manager.go

```diff
@@ -1269,3 +1277,11 @@ func fetchM3IPClaim(ctx context.Context, cl client.Client, mLog logr.Logger,
 	}
 	return metal3IPClaim, nil
 }
+
+// BuildProviderIDToNodes generates providerID and providerLabel by combining bmhuid and m3muid.
```

Comment:
Based on the func, it does return only providerUID, but as I read the comment am expecting two things as output? maybe it needs some rewording

Reply:
I agree. updated.